client/coral-sign-in/components/FakeComment.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import styles from 'coral-embed-stream/src/Comment.css';
+
+import AuthorName from 'coral-plugin-author-name/AuthorName';
+import Content from 'coral-plugin-commentcontent/CommentContent';
+import PubDate from 'coral-plugin-pubdate/PubDate';
+import {ReplyButton} from 'coral-plugin-replies';
+
+import I18n from 'coral-framework/modules/i18n/i18n';
+import translations from '../translations';
+
+const lang = new I18n(translations);
+
+class FakeComment extends React.Component {
+  constructor (props) {
+    super(props);
+  }
+
+  render () {
+    const {username, created_at, body} = this.props;
+
+    return (
+      <div
+        className={`comment ${styles.Comment}`}
```

Comment:
@gabelula I believe that this made a lot of the e2e tests fail. But folks might not have seen it. A lot of the '@element's in embedPageStream start their selectors with '.comment ...'. This matches the first one on the page, which this Dialog ends up being.

idk how the e2e tests do when they all run together, since they don't each clean up after themselves, but you can reproduce by running just this test on an empty db.

```
npm run pree2e && NODE_ENV=test ./node_modules/.bin/nightwatch --test test/e2e/tests/Commenter/LikeCommentTest.js
```

I'm working on a commit that will make the selectors specific to comments in the Embed Stream. I need it to make my e2e tests pass.